spec/features/review_tests/owner_emailed_for_new_review_spec.rb

```diff
@@ -0,0 +1,42 @@
+require "rails_helper"
+
+feature "owner receives email for new review", %{
+  As the owner of a created food truck
+  I want to receive an email when my truck is reviewed
+  So that I know what users think of it
+} do
+  # Acceptance Criteria
+  # If a food truck is reviewed, the creator of the food truck
+  # will be emailed.
+
+  scenario "food truck entry owner is emailed when new review is posted" do
+    owner = FactoryGirl.create(:user)
+    truck = FactoryGirl.create(:food_truck)
+    truck.user = owner
+    user = FactoryGirl.create(:user)
+
+    visit new_user_session_path
+
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
+
+    click_button 'Log in'
+
+    visit food_truck_path(truck)
+
+    expect(page).to have_content("Submit Your Review")
+
+    click_button('Submit Your Review')
+
+    expect(page).to have_content("Header")
```

Comment:
Pretty sure you already have tests that cover whether or not there's a form here, so maybe these expect statements aren't necessary for what this particular test wants to check for.